apps/storybook.namekit.io/stories/Namekit/Tooltip.stories.tsx

```diff
@@ -1,24 +1,15 @@
 import React from "react";
 import { Meta, StoryObj } from "@storybook/react";
-import {
-  Tooltip,
-  TooltipPlacement,
-  DEFAULT_MAX_TOOLTIP_WIDTH,
-} from "@namehash/namekit-react/client";
+import { Tooltip, TooltipPlacement } from "@namehash/namekit-react/client";
 
 const meta: Meta<typeof Tooltip> = {
   title: "UI/Tooltip",
   component: Tooltip,
+  tags: ["autodocs"],
   parameters: {
     layout: "centered",
   },
   argTypes: {
-    trigger: {
```

Comment:
It's better to totally hide these `argTypes`. We should only define `argTypes` for args that are proper for a user to directly manipulate in Storybook controls.

Before, it looked like this:

![CleanShot 2024-09-18 at 23 33 29](https://github.com/user-attachments/assets/a15a2c40-f11b-4e4e-b78a-a15b2267bb77)

Now it looks like this:

![CleanShot 2024-09-18 at 23 34 17](https://github.com/user-attachments/assets/159671e0-3cb5-4803-b5fe-0c18296c2086)